### DISCUSSION TITLE
MMDの規格(pmxフォーマット)への対応希望

### DISCUSSION THREAD

**Discussion by Water-blue-madao** [mika-f/blender-drag-and-drop#92] [Ideas]
素晴らしいアドオンを提供頂きありがとうございます！背景などのアセットの用意が容易になり、とても重宝しております！

　個人的な需要なのですが、MMDのモデル規格であるpmxフォーマットを、(必要であれば最もメジャーであろう[mmd_toolsアドオン](https://mmd-blender.fandom.com/ja/wiki/MMD_Tools)と併用することで)ドラッグ＆ドロップできるようになれば嬉しいです。特にMMDではデフォルトでドラッグ＆ドロップがサポートされているため、多少なりのギャップを感じておりました。

　mmd_toolsの方が現状3.6LTSをメインにサポートしているので今は良い機では無いのかも知れませんが、将来的な追加の検討をお願い申し上げます。

**Comment by mika-f** [comment]
よいアイデアだと思います！
[MMD Tools アドオン](https://mmd-blender.fandom.com/ja/wiki/MMD_Tools) を使う前提でしたら、追加は可能かと思われます。
